ffd_info_exchange/fafsa/forms.py

```diff
@@ -149,3 +160,89 @@ class FAFSAApplicationForm10(forms.Form):
     value_accounts = forms.IntegerField(label="As of today, how much money do your parents have in cash, savings accounts, and checking accounts?", min_value=0)
     value_investments = forms.IntegerField(label="As of today, what is the net worth of your parents’ investments, including real estate?", help_text="Don’t include the value of their home.", min_value=0)
     value_businesses = forms.IntegerField(label="As of today, what is the net worth of your parents’ current businesses and/or investment farms?", help_text="Don't include a family farm or family business with 100 or fewer full-time employees.", min_value=0)
+
+
+class FAFSAApplicationForm11(forms.Form):
+    # "Student tax information" section
+    # Thanks for sharing your parents’ financial information. Now we have a
+    # few questions about your tax information.
+    # ^ @todo: Consider making this phrasing more accommodating of 'parent'.
+    student_taxes_completed = forms.ChoiceField(choices=TAX_COMPLETION_STATUS_STUDENT, label="For 2015, have you completed your IRS income tax return or another tax return?")
+    # @todo: At some point, make the following dependent on the former. If they
+    # won't file or haven't filed, presumably there's nothing to pull.
+    consent_to_retrieve_data = forms.ChoiceField(choices=CONSENT_TO_RETRIEVE_DATA, label="Would you like to transfer your 2015 tax data from the IRS?", help_text="Allowing the IRS to transfer your tax information saves time and is more accurate than manually completing this section. The Department of Education will not store any of your tax information after your FAFSA is processed.")
```

Comment:
@kategarklavs I made up this label text because I didn't see a prompt. I was trying to avoid words like "share" because they run contrary to Liz Goodman's suggestions from login.gov, which emphasized onetime words like "transfer" instead.

Have at it! :) It's also possible that the options could stand on their own without a label.